reference/catalog/products_catalog.v3.yml

```diff
@@ -7289,11 +7281,6 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/productVariant_Full'
-      required:
```

Comment:
@bc-traciporter does this update mean that the `required` label will be gone for the Create Product sections?

Reply:
Hi @bc-romanliukshyn Yes, I added the requirement in the description field since the schema is shared by both the Create Products and Update Products endpoint.  Having the required tag caused confusion with the Update Product endpoint since it is not needed.

Reply:
Let me know if you have a better solution? We could always create a separate schema if you think that is a better solution.

Reply:
In my opinion, the `required` label serves an important purpose and enhances readability more effectively than a comment beside the field. Therefore, I believe we should keep it, even if it requires additional effort to restructure the schema.

Reply:
OK, no problem. I will create separate schemas for POST and PUT requests.